android/app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+
+    <application
+        android:usesCleartextTraffic="true"
```

Comment:
This is standard in ReactNative. It was also already like this before. Do we need to disable this for security reasons? I assume that the SDK performs all requests securely regardless of this setting.

Reply:
True, it doesn't affect sdk requests. It just won't block any other requests that are not secure. We don't have such requests so this comment is just to stick with good practices regarding security. If it can work with cleartext traffic disabled then we should remove that attribute, but if this is required by React Natvie then we can leave it.

Reply:
This is required by ReactNative for development as it sends the js bundle over http to the android app. Please note that this is disabled by default in android since api 28 and this manifest is exclusively for debug builds. The AndroidManifest.xml in the src/main does not contain this line.